src/Symfony/Component/Debug/ExceptionHandler.php

```diff
@@ -411,4 +414,27 @@ private function getSymfonyGhostAsSvg()
     }
+
+    private static function getTraceClassname($trace)
+    {
+        if (empty($trace['file'])) {
+            return null;
+        }
+
+        $vendorRoot = static::getVendorRoot();
+        $traceFileRelative = str_replace($vendorRoot, '', realpath($trace['file']));
+
+        if (strpos($traceFileRelative, '/vendor/') === 0) {
```

Comment:
Hm... How much this method is better than the currently implemented? I mean, I have based the vendor directory on the `ClassLoader` class from composer, and it is more simple than `LinkStub::getComposerRoot()` method. And just copy/paste don't solve the problem, I should adapt the code anyway.